### ISSUE TITLE
Hide the "None" category in Library grouping when all games are categorized

### ISSUE THREAD

**Issue by TheDiggleGod** [JosefNemec/Playnite#2124]
I have all my games grouped into categories by Source. Despite that, "None" still shows at the top of the list, even though every game has a source set in the metadata. It's a small thing, but it would be nice if the "none" category only showed up when there are games that aren't labeled under that tag. If everything has a tag under that sorting selection, then the "None" option just wouldn't exist. As it is right now, it just takes up space if you have everything properly categorized. 

Here's a picture of my sorting. You can see "None" at the top.
![image](https://user-images.githubusercontent.com/60529987/99597335-d6902880-29c5-11eb-9d38-b307f9dd70f3.png)

I found [this reddit post from a year ago](https://www.reddit.com/r/playnite/comments/cigfvw/none_category/), and a developer chimed in saying "It's not possible in the current version." I was wondering if there was any plans to make this change or if you guys just don't really care about something this small.

_I think I explained my point well, but let me know if I need to explain my point further._

**Comment by JosefNemec** [comment]
I don't think it makes sense to hide it. You would not be able to filter out games that have no source assigned.

**Comment by TheDiggleGod** [comment]
I tried to make it clear in my post, maybe I didn't do it very well, but I would want it to be there if there are actual games that aren't categorized. But if every single game in the whole library is in a category, then it wouldn't appear because it's showing nothing, like in the screenshot I posted.

Does that make sense?

**Comment by JosefNemec** [comment]
Problem with that is that it would affect performance. We would have to go through all games in the library when loading explorer lists.

**Comment by TheDiggleGod** [comment]
Okay, I just noticed this is already being done in Details View (list on the right). I knew that before, but I started using Grid View (and Fullscreen mode) recently, and it includes "All" and "None" like in the Explorer Panel (list on the left).
![image](https://user-images.githubusercontent.com/60529987/99701528-9337c780-2a62-11eb-9a58-4b3430672a80.png)

I think right now, my bigger problems are with Fullscreen Mode. So I might make another feature request regarding Fullscreen mode, rather than go after this smaller thing. 

Thank you for your time. :)

**Comment by NiceGlassesDude** [comment]
> Problem with that is that it would affect performance. We would have to go through all games in the library when loading explorer lists.

It should be possible to hide them at least. Another Platform I've used before did the same thing.

**Comment by 5ubie** [comment]
> It should be possible to hide them at least.

Seconding this, especially when using category and tag groupings in the explorer panel, it would be really nice to at least have a toggle in settings to hide either of the 2 groups if having a dynamic setting isn't realistic.
